pyproject.toml

```diff
@@ -97,7 +97,8 @@ pydist_resource_paths = ["my-package/img1.png", "my-package/foo/bar.json"]
 [tool.jupyter-releaser.hooks]
 before-build-npm = [
     "python -m pip install 'jupyterlab>=4.0.0a32'",
-    "jlpm"
+    "jlpm",
+    "jlpm build:prod"
```

Comment:
This is the fix :crossed_fingers:

Reply:
You also need to run `lerna build` at the top level to produce the `lib` files I believe.

Reply:
I'm running the command `jlpm build:prod` which runs `lerna run build:prod` at the top level but somehow is not building the packages. It says it ran successfully for all the packages but the lib folder isn't there.

Reply:
You would need a `build:prod` in each of the sub-packages for that to work.  You could run `build` and then `build:prod` I believe, or add `build:prod` scripts to each package.

Reply:
Thanks @blink1073